src/Cli/Microsoft.DotNet.Cli.Utils/Extensions/ArgumentExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+using System.Reflection;
+
+namespace Microsoft.DotNet.Cli.Utils.Extensions;
+
+public static class ArgumentExtensions
+{
+    private static readonly PropertyInfo[] s_nonPublicProperties = typeof(Argument).GetProperties(BindingFlags.Instance | BindingFlags.NonPublic);
+
+    public static bool? GetHasValidators(this Argument argument) =>
+        s_nonPublicProperties.First(pi => pi.Name == "HasValidators").GetValue(argument) as bool?;
```

Comment:
This was used when I was gathering Validators information. I'm no longer containing that information in the JSON, so this is no longer called. If y'all want me to delete this file, I can.

Reply:
Similarly, you'll see other extension classes having Validator methods. Those can be removed too.

Reply:
Good call - none of the Command/Argument extensions are used anymore, so they can be removed.